src/get/getPerceivedDifference.ts

```diff
@@ -0,0 +1,123 @@
+import { rad2deg, deg2rad } from "../constants";
+import { LabaColor } from "../types";
+
+/**
+ * kl - grafic arts = 1; textiles = 2;
+ * kl - unity factor;
+ * kh - weighting factor;
+ */
+interface Options {
+  kl: 1 | 2;
+  kc: number;
+  kh: number;
+}
+
+/**
+ * Calculates the perceived color difference according to [Delta E2000](https://en.wikipedia.org/wiki/Color_difference#CIEDE2000).
+ *
+ * ΔE - (Delta E, dE) The measure of change in visual perception of two given colors.
+ *
+ * Delta E is a metric for understanding how the human eye perceives color difference.
+ * The term delta comes from mathematics, meaning change in a variable or function.
+ * The suffix E references the German word Empfindung, which broadly means sensation.
+ *
+ * On a typical scale, the Delta E value will range from 0 to 100.
+ *
+ * | Delta E | Perception                             |
+ * |---------|----------------------------------------|
+ * | <= 1.0	 | Not perceptible by human eyes          |
+ * | 1 - 2	 | Perceptible through close observation  |
+ * | 2 - 10	 | Perceptible at a glance                |
+ * | 11 - 49 | Colors are more similar than opposite  |
+ * | 100	   | Colors are exact opposite              |
```

Comment:
In this library, I'm trying to normalize most of the values to fit `[0, 1]` ranges to make all methods more consistent.
`[0, 1]` is a more developer-friendly result in my opinion.

Reply:
Reasonable. The funny thing is, the `DeltaE 2000` sometimes can be equal more than 100. Should we clamp it in range [0, 1]?

Reply:
Yep, I think we should